pypeit/core/coadd.py

```diff
@@ -1294,67 +1294,234 @@ def order_phot_scale(spectra, phot_scale_dicts, nsig=3.0, niter=5, debug=False):
 
     return collate(spectra_list_new)
 
-def order_median_scale(spectra, nsig=3.0, niter=5, overlapfrac=0.03, num_min_pixels=50, SN_MIN_MEDSCALE=5.0, debug=False):
+def order_median_scale(wave, wave_mask, fluxes_in, ivar_in, sigrej=3.0, niter=5, min_overlap_pix=21, min_overlap_frac=0.03,
```

Comment:
Let's make lines of code no longer than 100 characters, and doc strings no wider than 72 characters.

Reply:
I'm fine with that but I don't know of a way to count characters on a line. My development IDE PyCharm does not do this for me.

Reply:
It seems like there should be a way to set a limit according to this:

https://stackoverflow.com/questions/17319422/how-do-i-set-the-maximum-line-length-in-pycharm

at least for the 100 character length.  May not be possible to do this and the doc string limitation.